docs/examples/prometheus_exemplars/docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+services:
+  # OpenTelemetry collector. Make sure you set USERID and GOOGLE_APPLICATION_CREDENTIALS
+  # environment variables for your container to authenticate correctly
+  otel-collector:
+    user: ${USERID?set USERID=$(id -u)}
+    image: otel/opentelemetry-collector-contrib:0.77.0
+    environment:
+      - GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS?}
+      - PROJECT_ID=${PROJECT_ID?}
+    volumes:
+      - ./otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml
+      - ${GOOGLE_APPLICATION_CREDENTIALS}:${GOOGLE_APPLICATION_CREDENTIALS}
+    ports:
+      - 4317:4317 # OTLP gRPC receiver
+
+  # runs server.py
+  python-server:
+    build:
+      context: .
+      dockerfile: server.dockerfile
+    init: true
+    ports:
+      - 6000:6000
+  
+  # load generator that sends requests to the python server
```

Comment:
Cool - I didn't know about this load generator before. I've used ApacheBench for similar situations.